### PR TITLE
Adds 3 new Polish foods

### PR DESCRIPTION
**Pull request by MidoriWroth**
## About The Pull Request

This PR will add 3 new foods to the game: pierogies, zurek, and stuffed cabbage. I plan to add more traditional Polish foods in the future, but I am pretty new to spriting and making things for SS13 so this is just an introductory effort for me.

## Why It's Good For The Game

Adding more food options to the game allows for more food diversity for chefs, and allows for more creative menus. I have always wanted to see more traditional human foods added to the game, so this is a small start.

## Changelog

:cl:
add: Added 3 new Polish foods
imageadd: Added sprites for the 3 new foods
/:cl:

### CODE REVIEW

**Comment by Spc-Dragonfruits** [comment]
pierogi in ss13 we're living in the future

File: code/game/objects/items/food/misc.dm [hunk 1]
```diff
@@ -963,3 +963,22 @@
 	icon_state = "ready_donk_warm_mex"
 	tastes = list("enchiladas" = 2, "laziness" = 1)
 	foodtypes = GRAIN | DAIRY | MEAT | VEGETABLES | JUNKFOOD
+
+/obj/item/food/pierogi
+	name = "pierogi"
+	desc = "A Polish dumpling made by wrapping unleavened dough around a savoury or sweet filling and cooking in boiling water. This one is filled with a potato and onion mixture."
```
Comment (Farquaar) [inline code/game/objects/items/food/misc.dm:969]: Pierogies are popular across Eastern Europe. They are not merely a Polish food, but an Eastern European food.

Comment (MidoriWroth) [reply code/game/objects/items/food/misc.dm:969]: Ah yeah, true!

**Comment by Colovorat** [comment]
i would argue that it's polish food, more like slavic, but whatever

**Review by LemonInTheDark — APPROVED** [review]
Code looks fine

**Pull request merged**